circle.yml

```diff
@@ -908,6 +908,16 @@ jobs:
           working_directory: npm/webpack-preprocessor/examples/react-app
       - store-npm-logs
 
+  npm-webpack-dev-server:
```

Comment:
I believe this stage need to be mentioned in the `jobs` section in order to be run on CI.

Reply:
done!